src/Juvix/Library.hs

```diff
@@ -116,6 +117,13 @@ traverseM ::
   f (m a2)
 traverseM f = fmap join . traverse f
 
+foldA ::
+  (Applicative f, Foldable t, Monoid a) =>
+  (b -> f a) ->
+  t b -> f a
+foldA f = foldl' (\acc x -> liftA2 (<>) acc (f x)) (pure mempty)
```

Comment:
Is this function similar to foldmap?

```haskell
foldMap :: (Foldable t, Monoid m) => (a -> m) -> t a -> m
```

Reply:
Actually yeah, I got `fold` and `foldMap` mixed up.

I was going to rename this to `foldMapA` but I forgot, sorry.